pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -113,6 +116,7 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 
 	// Set HyperConverged instance as the owner and controller
 	if err := controllerutil.SetControllerReference(instance, kvConfig, r.scheme); err != nil {
+		instance.Status.Phase = hcov1alpha1.FailedPhase
```

Comment:
You'll have to write the status back with a `r.client.Status().Update(context.TODO(), instance)` if you want more than the in-memory version of the instance to have the Phase.